src/solidity-frontend/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# replace it with FetchContent of CMake v3.11
+# to automatically download nlohmann_json as a dependency at configure stage
+add_subdirectory(nlohmann_json)
```

Comment:
Move this dependency to here: https://github.com/esbmc/esbmc/blob/master/scripts/cmake/ExternalDependencies.cmake

Reply:
ACK

Reply:
Done